TestAssets/TestProjects/TestProjectToProjectDependencies/src/L21/project.json

```diff
@@ -1,10 +1,10 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24018"
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
```

Comment:
Went ahead and implemented these changes. Required piping through the full framework name to the command result assertions that exist for testing whether build output actually compiled a project.